packaging/homebrew/README.md

```diff
@@ -0,0 +1,37 @@
+# Building a Homebrew package of the CLI
```

Comment:
copy pasted from shopify-app-cli. Wasn't particularly useful and am considering removing it. Thoughts?

Reply:
The rake commands look straightforward enough to me. Anything in this file that is not already automatically taken care of by the rake tasks?

Reply:
Nope yeah I'll remove this.